<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-11-08T16:56:46 IST" name="Suite" finished-at="2021-11-08T16:57:05 IST" duration-ms="18804">
    <groups>
    </groups>
    <test started-at="2021-11-08T16:56:46 IST" name="Test" finished-at="2021-11-08T16:56:58 IST" duration-ms="11975">
      <class name="com.ApertaWebApp_Automate.TestCases.TC_LoginTest_001">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.ApertaWebApp_Automate.TestCases.TC_LoginTest_001@59505b48]" started-at="2021-11-08T16:56:46 IST" name="setup" finished-at="2021-11-08T16:56:55 IST" duration-ms="8289" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="LoginTest()[pri:1, instance:com.ApertaWebApp_Automate.TestCases.TC_LoginTest_001@59505b48]" started-at="2021-11-08T16:56:55 IST" name="LoginTest" finished-at="2021-11-08T16:56:57 IST" duration-ms="2127" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginTest -->
        <test-method is-config="true" signature="close()[pri:0, instance:com.ApertaWebApp_Automate.TestCases.TC_LoginTest_001@59505b48]" started-at="2021-11-08T16:56:57 IST" name="close" finished-at="2021-11-08T16:56:58 IST" duration-ms="1480" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- com.ApertaWebApp_Automate.TestCases.TC_LoginTest_001 -->
    </test> <!-- Test -->
    <test started-at="2021-11-08T16:56:58 IST" name="Test1" finished-at="2021-11-08T16:57:05 IST" duration-ms="6685">
      <class name="com.ApertaWebApp_Automate.TestCases.TC_DCBTest_002">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.ApertaWebApp_Automate.TestCases.TC_DCBTest_002@20140db9]" started-at="2021-11-08T16:56:58 IST" name="setup" finished-at="2021-11-08T16:57:04 IST" duration-ms="5560" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="DCB()[pri:2, instance:com.ApertaWebApp_Automate.TestCases.TC_DCBTest_002@20140db9]" started-at="2021-11-08T16:57:04 IST" name="DCB" finished-at="2021-11-08T16:57:04 IST" duration-ms="10" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.ApertaWebApp_Automate.PageObjects.DCBPage.dcbtitle(DCBPage.java:20)
at com.ApertaWebApp_Automate.TestCases.TC_DCBTest_002.DCB(TC_DCBTest_002.java:30)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- DCB -->
        <test-method is-config="true" signature="close()[pri:0, instance:com.ApertaWebApp_Automate.TestCases.TC_DCBTest_002@20140db9]" started-at="2021-11-08T16:57:04 IST" name="close" finished-at="2021-11-08T16:57:05 IST" duration-ms="1107" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- com.ApertaWebApp_Automate.TestCases.TC_DCBTest_002 -->
    </test> <!-- Test1 -->
  </suite> <!-- Suite -->
</testng-results>
